pkgs/by-name/se/seabios/package.nix

```diff
@@ -64,9 +104,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     license = with lib.licenses; [ lgpl3Plus ];
     maintainers = with lib.maintainers; [ AndersonTorres ];
-    platforms = lib.systems.inspect.patternLogicalAnd
-      lib.systems.inspect.patterns.isUnix
-      lib.systems.inspect.patterns.isx86;
+    platforms = lib.systems.inspect.patternLogicalAnd lib.systems.inspect.patterns.isUnix lib.systems.inspect.patterns.isx86;
```

Comment:
```suggestion
    platforms = lib.intersectLists lib.platforms.unix lib.platforms.x86_64;
```

Reply:
String-y platforms are considered legacy (https://github.com/NixOS/nixpkgs/blob/fd698a4ab779fb7fb95425f1b56974ba9c2fa16c/lib/meta.nix#L208)

Reply:
Oh wow. Why is everyone still using it?

Reply:
I believe it is because the modern way is not so well documented and it is a bit harder to understand.

Further, there are some dark corners on Nixpkgs that still expect the stringy values.